scientific/src/commonMain/kotlin/converter/dimensionless/ConvertToAnyUnit.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.converter.dimensionless
+
+import com.splendo.kaluga.scientific.PhysicalQuantity
+import com.splendo.kaluga.scientific.ScientificValue
+import com.splendo.kaluga.scientific.div
+import com.splendo.kaluga.scientific.times
+import com.splendo.kaluga.scientific.unit.One
+import com.splendo.kaluga.scientific.unit.ScientificUnit
+import com.splendo.kaluga.scientific.unit.constant
+
+infix operator fun <
+    Quantity : PhysicalQuantity,
+    Unit : ScientificUnit<Quantity>,
+    Modifier : ScientificUnit<PhysicalQuantity.Dimensionless>
+    > ScientificValue<Quantity, Unit>.times(
+    modifier: ScientificValue<PhysicalQuantity.Dimensionless, Modifier>
+): ScientificValue<Quantity, Unit> {
```

Comment:
There should also be an implementation of these methods using a value factory (factory: (Decimal, Unit) -> Value), where Value : ScientificValue<Quantity, Unit>

In addition, you can use Unit.byDividing and Unit.byMultiplying so you dont have to do the conversion yourself:

```
infix operator fun <
    Quantity : PhysicalQuantity,
    Unit : ScientificUnit<Quantity>,
    Modifier : ScientificUnit<PhysicalQuantity.Dimensionless>,
    Value : ScientificValue<Quantity, Unit>
    > ScientificValue<Quantity, Unit>.times(
    modifier: ScientificValue<PhysicalQuantity.Dimensionless, Modifier>,
    factory: (Decimal, Unit) -> Value
): Value = unit.byMultiplying(this, modifier, factory)
```

Reply:
Also if Unit * Dimensionless = Unit, then Dimensionless * Unit = Unit, so you need operators for that too

Reply:
> There should also be an implementation of these methods using a value factory (factory: (Decimal, Unit) -> Value), where Value : ScientificValue<Quantity, Unit>
Can you make an operator with two arguments?

Reply:
No, it should just be a function and the operator should call it with the default factory. See other converters for an example